sql-plugin/src/main/scala/org/apache/spark/sql/rapids/mathExpressions.scala

```diff
@@ -47,8 +47,10 @@ case class GpuAcos(child: Expression) extends CudfUnaryMathExpression("ACOS") {
 case class GpuToDegrees(child: Expression) extends GpuUnaryMathExpression("DEGREES") {
 
   override def doColumnar(input: GpuColumnVector): ColumnVector = {
-    withResource(Scalar.fromDouble(180d / Math.PI)) { multiplier =>
-      input.getBase.mul(multiplier)
+    withResource(Scalar.fromDouble(180d)) { multiplier =>
+      withResource(Scalar.fromDouble(Math.PI)) { divisor =>
+        input.getBase.mul(multiplier).div(divisor)
+      }
```

Comment:
`input.getBase.mul(multiplier)` produces a resource that needs to be closed.
The `180d` scalar can be closed as soon as possible to release some memory. Here it's trivial but this is a good practice.


Suggests:
```
    val tmp = withResource(Scalar.fromDouble(180d)) { multiplier =>
      input.getBase.mul(multiplier)
    }

    // at here the 180d scalar is closed.
    withResource(tmp) { _ =>
      withResource(Scalar.fromDouble(Math.PI)) { pi =>
        tmp.div(pi)
      }
    }
```

Reply:
ok, revised.